src/Codeception/Lib/Connector/Laravel.php

```diff
@@ -463,4 +463,36 @@ public function clearApplicationHandlers(): void
     {
         $this->applicationHandlers = [];
     }
+    
+    /**
+     * Make sure files are \Illuminate\Http\UploadedFile instances with the private $test property set to true.
+     * Fixes issue https://github.com/Codeception/Codeception/pull/3417.
+     *
+     * @param array $files
+     * @return array
+     */
+    protected function filterFiles(array $files)
```

Comment:
```suggestion
    protected function filterFiles(array $files): array
```
type hints can also be removed here.

Reply:
I thought this was going to break since the parent method does not explicitly return array.